frontend/keysets/simple-urdu-keyset/simple-urdu-keys-layout.ts

```diff
@@ -0,0 +1,31 @@
+export const unshiftedKeysLayout = [
+  ["`", "۱", "۲", "۳", "۴", "۵", "٦", "۷", "۸", "۹", "۰", "-", "=", "Delete"],
+  ["Tab", "ق", "و", "ع", "ر", "ت", "ے", "ء", "ی", "ہ", "پ", "[", "]", "\\"],
+  ["Caps", "ا", "س", "د", "ف", "گ", "ح", "ج", "ک", "ل", "؛", "'", "Enter"],
+  ["Shift", "ز", "ش", "چ", "ط", "ب", "ن", "م", "،", "۔", "/", "Shift"],
+  ["Ctrl", "Alt", "Space", "Alt", "Ctrl"],
+];
+
+export const shiftedKeysLayout = [
+  ["~", "!", "@", "#", "٪", "ﷲ", "ٔ", "،", "*", ")", "(", "_", "+", "Delete"],
+  ["Tab", "ٰ", "ُ", "ٗ", "ڑ", "ٹ", "ۓ", "ّ", "َ", "ۂ", "ﷺ", "{", "}", "|"],
+  ["Caps", "آ", "ص", "ڈ", "", "غ", "ح", "ژ", "خ", "ُ", ":", "\"", "Enter"],
+  ["Shift", "ذ", "ً", "ٔ", "ظ", "ب", "ں", "؟", ">", "ّ", "؟", "Shift"],
+  ["Ctrl", "Alt", "Space", "Alt", "Ctrl"],
+];
+
+export const capsKeysLayout = [
+  ["`", "۱", "۲", "۳", "۴", "۵", "٦", "۷", "۸", "۹", "۰", "-", "=", "Delete"],
```

Comment:
```suggestion
  ["`", "۱", "۲", "۳", "۴", "۵", "۶", "۷", "۸", "۹", "۰", "ّ", "-", "=", "Delete"],
```
- use urdu `6`
- add key for diacritic